src/pools/D3MAavePool.sol

```diff
@@ -173,8 +181,11 @@ contract D3MAavePool is ID3MPool {
         require(dai.balanceOf(msg.sender) == prevDai + wad, "D3MAavePool/incorrect-dai-balance-received");
     }
 
-    function transfer(address dst, uint256 wad) external override onlyHub {
-        require(adai.transfer(dst, wad), "D3MAavePool/transfer-failed");
+    function exit(address dst, uint256 wad) external override onlyHub {
+        uint256 exited_ = exited;
+        exited = exited_ + wad;
+        uint256 amt = wad * assetBalance() / (D3mHubLike(hub).end().Art(ilk) - exited_);
```

Comment:
There's an implicit invariant here, right? That `end().Art(ilk)` needs to be at least as much as the exitable `ink`. Couldn't that be violated in some repayment scenarios?

Reply:
Not suggesting we need to block on it here, just calibrating my understanding for unit testing.